src/test/java/dev/openfga/sdk/api/client/StreamedListObjectsTest.java

```diff
@@ -1,426 +1,429 @@
-                .user(DEFAULT_USER);
-
-        // When
-        CompletableFuture<Void> future = fga.streamedListObjects(request, null, obj -> {}, receivedErrors::add);
-
-        try {
-            future.get();
-            fail("Expected exception");
-        } catch (Exception e) {
-            // Expected to fail
-        }
-
-        // Then - verify the error consumer received the original ApiException, not wrapped
-        assertEquals(1, receivedErrors.size());
-        Throwable error = receivedErrors.get(0);
-        assertTrue(
-                error instanceof dev.openfga.sdk.errors.ApiException,
-                "Expected ApiException but got " + error.getClass().getName());
-        dev.openfga.sdk.errors.ApiException apiException = (dev.openfga.sdk.errors.ApiException) error;
-        assertEquals(400, apiException.getStatusCode());
-    }
-
-    private HttpResponse<Stream<String>> createMockStreamResponse(int statusCode, Stream<String> body) {
-        HttpResponse<Stream<String>> mockResponse = mock(HttpResponse.class);
-        when(mockResponse.statusCode()).thenReturn(statusCode);
-        when(mockResponse.body()).thenReturn(body);
-
-        // Create mock headers
-        HttpHeaders mockHeaders = mock(HttpHeaders.class);
-        when(mockHeaders.map()).thenReturn(Map.of("content-type", List.of("application/json")));
-        when(mockResponse.headers()).thenReturn(mockHeaders);
-
-        return mockResponse;
-    }
-}
+// NOTE: This test is temporarily commented out as the streamedListObjects API is not yet available for public use.
+// It will be enabled in a future release.
+
+// package dev.openfga.sdk.api.client;
+//
+// import static org.junit.jupiter.api.Assertions.*;
+// import static org.mockito.ArgumentMatchers.any;
+// import static org.mockito.Mockito.*;
+//
+// import com.fasterxml.jackson.databind.ObjectMapper;
+// import dev.openfga.sdk.api.client.model.ClientListObjectsRequest;
+// import dev.openfga.sdk.api.client.model.ClientStreamedListObjectsOptions;
+// import dev.openfga.sdk.api.configuration.ClientConfiguration;
+// import dev.openfga.sdk.api.configuration.Credentials;
+// import dev.openfga.sdk.api.model.ConsistencyPreference;
+// import dev.openfga.sdk.api.model.StreamedListObjectsResponse;
+// import dev.openfga.sdk.constants.FgaConstants;
+// import dev.openfga.sdk.errors.FgaInvalidParameterException;
+// import java.net.http.HttpClient;
+// import java.net.http.HttpHeaders;
+// import java.net.http.HttpResponse;
+// import java.time.Duration;
+// import java.util.ArrayList;
+// import java.util.List;
+// import java.util.Map;
+// import java.util.concurrent.CompletableFuture;
+// import java.util.concurrent.atomic.AtomicInteger;
+// import java.util.stream.Stream;
+// import org.junit.jupiter.api.BeforeEach;
+// import org.junit.jupiter.api.Test;
+//
+/// ** Tests for streaming list objects functionality with CompletableFuture. */
+// public class StreamedListObjectsTest {
```

Comment:
Could we annotate the class with a [Disabled](https://docs.junit.org/5.3.1/api/org/junit/jupiter/api/Disabled.html) annotation for now rather than commenting out?

```suggestion
@Disabled("streamedListObjects is private for now")
// public class StreamedListObjectsTest {
```

Reply:
Oh yes that's probably a better way to do this